grease/src/Grease/Macaw/Load.hs

```diff
@@ -313,10 +325,10 @@ resolveCoreDumpEntrypointAddress ::
   -- | A 'Map.Map' of function entrypoint addresses to their names.
   Map.Map (MM.MemSegmentOff w) BS.ByteString ->
   -- | The path to the core dump file, which was obtained by running the binary
-  -- to simulate.
+  --     to simulate.
```

Comment:
Huh, is there an option to control this? Don't love the extra spaces here. nbd though.